lib/pure/net.nim

```diff
@@ -36,8 +37,16 @@
 ##   var socket = newSocket()
 ##   socket.connect("google.com", Port(80))
 ##
+## For SSL, use the next example.
+##
+## .. code-block:: Nim
+##   var socket = newSocket()
+##   var ctx = newContext()
+##   wrapSocket(ctx, socket)
+##   socket.connect("google.com", Port(443))
+##
 ## UDP is a connectionless protocol, so UDP sockets don't have to explicitly
-## call the ``connect`` procedure. They can simply start sending data
+## call the `connect` procedure. They can simply start sending data
```

Comment:
Doesn't this need 2 backticks? Otherwise it's just indented.

Reply:
Nim doc uses a RST/MD hybrid parser that can handle this, though it'd be better to link to the `proc` directly instead.

Reply:
I used single backticks to comply with the Contributor Guidelines. As suggested I changed the `connect` and `newContext` references to internal links.